docs/src/adoc/postgresql.adoc

```diff
@@ -1 +1,15 @@
-TODO
+*jdbi3-postgres* provides enhanced integration with the 
+link:https://jdbc.postgresql.org/[PostgreSQL JDBC Driver].
+
+[source,java,indent=0]
```

Comment:
Heads up: the `indent=0` is only necessary when including a tagged from another file. Inline code doesn't need it.